pkg/cmd/cli/describe/describer.go

```diff
@@ -368,6 +372,124 @@ func (d *ProjectDescriber) Describe(namespace, name string) (string, error) {
 	})
 }
 
+// ProjectStatusDescriber generates extended information about a Project
+type ProjectStatusDescriber struct {
+	K kclient.Interface
+	C client.Interface
+}
+
+func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error) {
+	project, err := d.C.Projects().Get(namespace)
+	if err != nil {
+		return "", err
+	}
+
+	svcs, err := d.K.Services(namespace).List(labels.Everything())
+	if err != nil {
+		return "", err
+	}
+
+	bcs, err := d.C.BuildConfigs(namespace).List(labels.Everything(), fields.Everything())
+	if err != nil {
+		return "", err
+	}
+
+	dcs, err := d.C.DeploymentConfigs(namespace).List(labels.Everything(), fields.Everything())
+	if err != nil {
+		return "", err
+	}
+
+	g := graph.New()
+	for i := range bcs.Items {
+		graph.BuildConfig(g, &bcs.Items[i])
+	}
+	for i := range dcs.Items {
+		graph.DeploymentConfig(g, &dcs.Items[i])
+	}
+	for i := range svcs.Items {
+		graph.Service(g, &svcs.Items[i])
+	}
+	groups := graph.ServiceAndDeploymentGroups(graph.CoverServices(g))
+
+	return tabbedString(func(out *tabwriter.Writer) error {
+		indent := "  "
+		if len(project.DisplayName) > 0 && project.DisplayName != namespace {
+			fmt.Fprintf(out, "In project %s (%s)\n", project.DisplayName, namespace)
```

Comment:
Feels like clubbing the user over the head with the word status.  Tried to be consistent with 'git status' ("On branch master") as the first line

----- Original Message -----
> > +	g := graph.New()
> > +	for i := range bcs.Items {
> > +		graph.BuildConfig(g, &bcs.Items[i])
> > +	}
> > +	for i := range dcs.Items {
> > +		graph.DeploymentConfig(g, &dcs.Items[i])
> > +	}
> > +	for i := range svcs.Items {
> > +		graph.Service(g, &svcs.Items[i])
> > +	}
> > +	groups := graph.ServiceAndDeploymentGroups(graph.CoverServices(g))
> > +
> > +	return tabbedString(func(out *tabwriter.Writer) error {
> > +		indent := "  "
> > +		if len(project.DisplayName) > 0 && project.DisplayName != namespace {
> > +			fmt.Fprintf(out, "In project %s (%s)\n", project.DisplayName,
> > namespace)
> 
> maybe "Status of project foo"
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1314/files#r27436461